config.yml

```diff
@@ -1098,6 +1098,18 @@ nodes:
           def a(*b)
                 ^^
           end
+  - name: ArraySplatNode
```

Comment:
Can we call this a `StarNode`? It's going to be used in more places than just arrays, as in:

```ruby
foo(*bar)
foo => [*bar, baz]
foo, *bar = baz
```